### PR TITLE
Extend README

### PR DESCRIPTION
**Pull request by miiu96**
- Extend the readme file
- Added `Dockerfile` for the indexer binary

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+FROM golang:1.17.6 as builder
```
Comment (sstanculeanu) [inline Dockerfile:1]: shouldn't this be 1.20?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,148 @@
...
+Response: Metrics are formatted in a way that Prometheus can scrape and ingest for monitoring and alerting purposes.
+
+
+
+### Prerequisites
+Before proceeding, ensure you have the following prerequisites:
+- Go programming environment set up.
+- Access to an Elasticsearch database instance.
+- One has to setup one or multiple observers. For running an observing squad, these [docs](https://docs.multiversx.com/integrators/observing-squad/) cover the whole process.
+The required configs for launching an observer/s with a driver attached, can be found here.
```
Comment (sstanculeanu) [inline README.md:58]: missing link on here?

File: Dockerfile (hunk 1 above)
Comment (miiu96) [reply Dockerfile:1]: changed

File: README.md (hunk 2 above)
Comment (miiu96) [reply README.md]: done

**Review by sstanculeanu — APPROVED** [review]

**Review by schimih — APPROVED** [review]

**Pull request merged**